driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/ODBFeatures.java

```diff
@@ -107,7 +108,7 @@ public boolean supportsComputer() {
 
         @Override
         public boolean supportsTransactions() {
-            return false;
+            return transactionalGraph;
```

Comment:
If we decide to implement it that way we should get rid of getNoTx() or add a configuration flag which is used to determine the transactional graph behaviour.